io/include/pcl/io/openni2/openni2_device_manager.h

```diff
@@ -56,16 +57,16 @@ namespace pcl
 
         // This may not actually be a singleton yet. Need to work out cross-dll incerface.
         // Based on http://stackoverflow.com/a/13431981/1789618
-        static boost::shared_ptr<OpenNI2DeviceManager> getInstance ()
+        static std::shared_ptr<OpenNI2DeviceManager> getInstance ()
         {
-          static boost::shared_ptr<OpenNI2DeviceManager> instance = boost::make_shared<OpenNI2DeviceManager>();
+          static auto instance = std::make_shared<OpenNI2DeviceManager>();
```

Comment:
Self: Revise later if replaced by `pcl::make_shared`.

Reply:
I'll switch to `pcl::make_shared`. I think our policy should be "always use pcl::make_shared, no matter if there is a custom allocator or not" .